workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO;
+using System.Text;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public static class LaunchMenu
+    {
+        private const string rootApkPath = "build";
+        private const string MenuLaunchAndroid = "SpatialOS/Launch mobile client/Android Device";
+
+        [MenuItem(MenuLaunchAndroid, false, 73)]
+        private static void LaunchMobileClient()
+        {
+            try
+            {
+                EditorUtility.DisplayProgressBar("Launching Mobile Client", "Installing APK", 0.3f);
+
+                // Find apk to install
+                var rootPath = Path.Combine(Application.dataPath, Path.Combine("..", rootApkPath));
+                if (!TryGetApkPath(rootPath, out var apkPath))
+                {
+                    Debug.LogError($"Could not find a built out Android binary in \"{rootPath}\" to launch.");
+                    return;
+                }
+
+                // Install apk on connected phone / emulator
+                RedirectedProcess.Run("adb", "install", "-r", apkPath);
```

Comment:
still no check if adb actually exists.

Reply:
Correct, rather out of scope of this initial implementation

Reply:
so what happens right now, if someone hasn't adb installed? cause this will troll ltierally everyone that sees that button and just wants to quickly check out android

Reply:
plus I'm not sure if that will work on mac. I know that we had to use `osascript` for it to work

Reply:
Yeah, this will most likely fail on Mac. Should we just show error that adb has to be installed on mac? Installing it here might be too convoluted. https://stackoverflow.com/questions/17901692/set-up-adb-on-mac-os-x describes common install ways - none of them look simple enough, though we might download and install it if we really want to